state-chain/runtime/src/lib.rs

```diff
@@ -1324,6 +1427,7 @@ impl_runtime_apis! {
 				ForeignChainAndAsset::Polkadot(asset) => MinimumDeposit::<Runtime, PolkadotInstance>::get(asset),
 				ForeignChainAndAsset::Bitcoin(asset) => MinimumDeposit::<Runtime, BitcoinInstance>::get(asset).into(),
 				ForeignChainAndAsset::Arbitrum(asset) => MinimumDeposit::<Runtime, ArbitrumInstance>::get(asset),
+				ForeignChainAndAsset::Solana(asset) => MinimumDeposit::<Runtime, SolanaInstance>::get(asset),
```

Comment:
Probably needs to be set somewhere? Would be good to have a list of pallet configuration items that need to be set for new chains...

Reply:
This has to be set via governance extrinsic ```update_pallet_config``` in the ingress egress pallet.

Reply:
Can we maintain a list of these tasks please, so that we don't forget?

Reply:
sure, I will try to put together a list.

Reply:
created a notion doc for chain initialization realted tasks:

https://www.notion.so/chainflip/New-chain-initialization-todo-list-6c03c525fc9a455297e67c6d37c8bbf1

please add if anything else we need to do for new chains.